Obleak.Fody/Extensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Cecil.Rocks;
+
+namespace Obleak.Fody
+{
+    internal static class Extensions
```

Comment:
personally prefer classes in own file, but I don't think I have the authority to mandate coding style in this case :laughing: